ecommerce/migrations/0004_remove_productsubcategory_ecommerce_p_name_9e1b5d_idx_and_more.py

```
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.23 on 2025-08-06 01:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ecommerce', '0003_alter_productcategory_options_and_more'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)